python/lsst/meas/deblender/sourceDeblendTask.py

```diff
@@ -353,14 +353,14 @@ def deblend(self, exposure, srcs, psf):
             if self.isLargeFootprint(fp):
                 src.set(self.tooBigKey, True)
                 self.skipParent(src, mi.getMask())
-                self.log.warning('Parent %i: skipping large footprint (area: %i)',
-                                 int(src.getId()), int(fp.getArea()))
+                self.log.debug('Parent %i: skipping large footprint (area: %i)',
+                               int(src.getId()), int(fp.getArea()))
                 continue
             if self.isMasked(fp, exposure.getMaskedImage().getMask()):
                 src.set(self.maskedKey, True)
                 self.skipParent(src, mi.getMask())
-                self.log.warning('Parent %i: skipping masked footprint (area: %i)',
-                                 int(src.getId()), int(fp.getArea()))
+                self.log.debug('Parent %i: skipping masked footprint (area: %i)',
```

Comment:
I do wish this one logged what masks caused it to be skipped, but that's not your problem.

Reply:
Yeah, I think that would require a rewrite/rename/replacement of `isMasked`.